.github/workflows/release.yml

```diff
@@ -62,9 +70,12 @@ jobs:
 
       # Dockerhub is auto synced with the repo, no need to explicitly deploy
       - name: build and push latest docker image to GCR
-        run: |
-          docker build -t ghcr.io/codfish/semantic-release-action:latest .
-          docker push ghcr.io/codfish/semantic-release-action:latest
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          platforms: linux/amd64,linux/amd64
```

Comment:
you mean to have arm here as well?

Reply:
d'oh! definitely not - just pushed a commit to use arm64 there.